src/core/lib/security/credentials/s2a/grpc_s2a_credentials_options.cc

```diff
@@ -25,13 +25,11 @@
 #include <grpc/support/string_util.h>
 #include <string.h>
 
-#include <iostream>
-
 namespace experimental {
 
 grpc_s2a_credentials_options::~grpc_s2a_credentials_options() {
   if (handshaker_service_url_ != nullptr) {
-    gpr_free(handshaker_service_url_);
+    gpr_free(const_cast<char*>(handshaker_service_url_));
```

Comment:
If we use a unique pointer for `handshaker_service_url_`, we probably don't need to cast.

Reply:
This now moot since we use `std::string`.